user_guide_src/source/installation/upgrade_views.rst

```diff
@@ -0,0 +1,78 @@
+Upgrade Views
+###############################################################################
+
+.. contents::
+    :local:
+    :depth: 1
+
+
+Documentations
+============================================================
+
+- `View Documentation Codeigniter 3.X <http://codeigniter.com/userguide3/general/views.html>`_
+- `View Documentation Codeigniter 4.X <http://codeigniter.com/user_guide/outgoing/views.html>`_
+
+
+What has been changed
+============================================================
+- Your views look much like before, but they are invoked differently … instead of CI3’s $this->load->view(x); you can use echo view(x);
+- CI4 supports view “cells”, to build your response in pieces
+- The template parser is still there, but substantially enhanced
```

Comment:
```suggestion
- Your views look much like before, but they are invoked differently … instead of CI3’s ``$this->load->view(x);``, you can use ``echo view(x);``.
- CI4 supports view “cells”, to build your response in pieces.
- The template parser is still there, and substantially enhanced.
```